src/Main.java

```diff
@@ -1,5 +1,45 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Task 1");
+        int[] expensesPerWeek = {23_500, 22_000, 24_000, 26_500, 21_200};
+        int sum = 0;
+        for (int i : expensesPerWeek) {
+            sum +=i;
+        }
+        System.out.printf("Сумма трат за месяц составила %d рублей.\n\n", sum);
+
+
+        System.out.println("Task 2");
+
+        int min = expensesPerWeek[1];
```

Comment:
Подход правильный, но у массива индексы начинаются с нуля
```suggestion
        int min = expensesPerWeek[0];
```